portal/server/app/route.ts

```diff
@@ -30,8 +31,20 @@ export async function GET(req: Request) {
     const requestDomain = getDomain(url)
 
     if (requestDomain == portalDomain && parsedUrl && parsedUrl.subdomain) {
-        console.log('fetching from the service worker')
-        return resolveAndFetchPage(parsedUrl)
+        const forwardToFallback = async () => {
+            const subdomain = parsedUrl.subdomain;
+            const fallbackDomain = process.env.FALLBACK_DEVNET_PORTAL;
+            return fetch(`https://${subdomain}.${fallbackDomain}${parsedUrl.path}`)
```

Comment:
Question: move to a separate function? or any specific reason it's in a closure?

Reply:
I preferred the closure since I could use outer scoped values inside it without creating function arguments, plus defining it the same place that it's used, keeping it simpler. 
It's also a short implementation and it will only be used temporarily, so when the time comes to remove it I will just delete this specific line range.